q2_types/feature_data/_transformer.py

```diff
@@ -227,6 +240,25 @@ def _read_dna_fasta(path):
     return skbio.read(path, format='fasta', constructor=skbio.DNA)
 
 
+def _dnafastaformats_to_series(ff):
+    data = {}
+    for sequence in _read_dna_fasta(str(ff)):
+        id_ = sequence.metadata['id']
+        if id_ in data:
+            raise ValueError("FASTA format feature IDs must be unique. The "
```

Comment:
Can you change "feature IDs" to "sequence IDs"? The latter is more generally applicable to the FASTA format (since we don't necessarily know that "feature" sequences are being loaded).